src/middleware/errorHandler.ts

```
@@ -16,7 +17,9 @@ export class AppError extends Error {
     super(message);
     this.statusCode = statusCode;
     this.code =
-      typeof codeOrDetails === "string" ? codeOrDetails : "APP_ERROR";
+      typeof codeOrDetails === "string"
+        ? codeOrDetails
+        : ErrorCodes.BAD_REQUEST;
```

ESLint

[error] Replace `⏎······typeof·codeOrDetails·===·"string"⏎········?·codeOrDetails⏎·······` with `·typeof·codeOrDetails·===·"string"·?·codeOrDetails`

(prettier/prettier)